Add doc about collector account minimum permssions (#18)
kubev2v/migration-planner: doc/permssions.md

> # Minimum permissions for collector account

Reviewer: `Collector` is not the common name I think 
Maybe @avishayt @machacekondra can help here?

Reviewer 2: Yeah, I wouldn't use the `collector` name as well. Maybe better is `agent` or something similar.

The **collector** only gathers information from vCenter, therefore a **read only** permission whould be enough. 

One such role is the [Read Only Role](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-93B962A7-93FA-4E96-B68F-AE66D3D6C663.html).

This permissions needs to be applied at the root object level with "Propagate to children" selected.

> Remark: The **Read Only** role is not sufficient for MTV operator.

## Credentials to login into vCenter

> In order for the **collector** to be able to login into vCenter, `username` and `password` must be supplied. 

Reviewer: To enable the **collector** (or another name we will decide) to log in to vCenter, the `username` and `password` must be provided.

> 
> In order for the **collector** to be able to login into vCenter, `username` and `password` must be supplied. 

Reviewer: ```suggestion

To enable the **collector** (or another name we will decide) to log in to vCenter, the `username` and `password` must be provided.

```

> Unfortunatly, a `token` is not an option due to limitation in golang [vSphere client](https://pkg.go.dev/github.com/vmware/govmomi#Client.Login).

Reviewer: ```suggestion
Unfortunately,  using a `vCenter token` is not supported due to a Golang limitation [vSphere client](https://pkg.go.dev/github.com/vmware/govmomi#Client.Login).
```
